src/tool_source/github.rs

```diff
@@ -18,20 +20,25 @@ const APP_NAME: &str = "LPGhatguy/aftman";
 
 pub struct GitHubSource {
     client: Client,
+    token: Option<String>,
 }
 
 impl GitHubSource {
-    pub fn new() -> Self {
+    pub fn new(home: &Home) -> Self {
+        let token = AuthManifest::load(home).ok();
         Self {
             client: Client::new(),
+            token: token.flatten().map(|t| t.github).flatten(),
         }
     }
 
     pub fn get_all_releases(&self, name: &ToolName) -> anyhow::Result<Vec<Release>> {
         let url = format!("https://api.github.com/repos/{}/releases", name);
-        let builder = self.client.get(&url).header(USER_AGENT, APP_NAME);
+        let mut builder = self.client.get(&url).header(USER_AGENT, APP_NAME);
 
-        // TODO: Authorization
+        if let Some(token) = &self.token {
+            builder = builder.header(AUTHORIZATION, format!("token {}", token));
+        }
```

Comment:
Don't forget to add this same code to the request to download an artifact. It's tagged with a "TODO: Authorization" later in this file.